source/lambda/testing_requirements.txt

```diff
@@ -1,7 +1,7 @@
 botocore>=1.12.99
```

Comment:
Why not lock all package versions?

Reply:
I tend to use `botocore~=1.12.99` so that new patch versions of libraries are automatically used without the need to change the library version in (testing-)requirements.txt.
If I fix the version with "==", every published patch version release will cause the Viperlight prebuild scan to fail, costing extra cycles. The annoyance outweighs the benefits imho.